libraries/AP_GPS/AP_GPS_SBP2.h

```diff
@@ -0,0 +1,184 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//
+//  Swift Navigation SBP GPS driver for ArduPilot.
+//	Code by Niels Joubert
+//
+//  Swift Binary Protocol format: http://docs.swift-nav.com/
+//
+#pragma once
+
+#include "AP_GPS.h"
+#include "GPS_Backend.h"
+
+class AP_GPS_SBP2 : public AP_GPS_Backend
+{
+public:
+    AP_GPS_SBP2(AP_GPS &_gps, AP_GPS::GPS_State &_state, AP_HAL::UARTDriver *_port);
+
+    AP_GPS::GPS_Status highest_supported_status(void) { return AP_GPS::GPS_OK_FIX_3D_RTK_FIXED; }
+
+    // Methods
+    bool read();
```

Comment:
I don't quite understand? I'm looking at the AP_GPS_UBLOX.h as an example, and I don't see an override mentioned on read() or on highest_supported_status(). Can you elaborate?